src/components/ResultCard/ResultCard.tsx

```diff
@@ -37,6 +42,15 @@ export default function ResultCard(props: ResultCardProps) {
       </div>
       <div className='cio-result-card-text'>
         <p className='cio-result-card-title'>{result.value}</p>
+        <div className='cio-result-card-rating'>
```

Comment:
This is not a must have, a question.
What do you think about putting rating and prices into a container div? That way if anyone wants to flip their position (price first), they can do so easily with CSS.

Reply:
Actually good idea!